packages/shared/src/lib/feed.ts

```diff
@@ -85,18 +85,18 @@ export function postAnalyticsEvent(
     feed_grid_columns: opts?.columns,
     feed_item_grid_column: opts?.column,
     feed_item_grid_row: opts?.row,
-    feed_item_image: post.image,
-    feed_item_target_url: post.permalink,
-    feed_item_title: post.title,
-    post_author_id: post.author?.id,
-    post_created_at: post.createdAt,
-    post_comments_count: post.numComments,
-    post_read_time: post.readTime,
-    post_tags: post.tags,
-    post_source_id: post.source?.id,
-    post_trending_value: post.trending,
-    post_upvotes_count: post.numUpvotes,
-    target_id: post.id,
+    feed_item_image: post?.image,
+    feed_item_target_url: post?.permalink,
+    feed_item_title: post?.title,
+    post_author_id: post?.author?.id,
+    post_created_at: post?.createdAt,
+    post_comments_count: post?.numComments,
+    post_read_time: post?.readTime,
+    post_tags: post?.tags,
+    post_source_id: post?.source?.id,
+    post_trending_value: post?.trending,
+    post_upvotes_count: post?.numUpvotes,
+    target_id: post?.id,
```

Comment:
Why is this needed? If `post` is not defined we better don't do anything in this case